.github/workflows/test.yml

```
@@ -27,123 +27,60 @@ jobs:
-        run: docker pull ${{ env.DOCKER_REGISTRY_PATH }}/${{ inputs.project_name }}-development:${{ inputs.project_version }}
-      - name: Download Python packages
-        uses: actions/download-artifact@v4
-        with:
-          name: python-bindings
-          path: packages/python/
-      - name: Run Unit Tests Python
-        run: make ci-test-python ${{ format('test_python_version={0}', matrix.python-version) }}
+      - name: Build Python Tests
+        run: cd /app && make _build-test-python
+      - name: Run Python Unit Tests
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2010:warning:1:18: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)